soh/src/code/main.c

```diff
@@ -42,7 +42,7 @@ void main(int argc, char** argv)
     InitOTR();
     BootCommands_Init();
 
-    BootCommands_ParseBootArgs(argc - 1, (char**)&argv[1]);
+    // BootCommands_ParseBootArgs(argc - 1, (char**)&argv[1]);
```

Comment:
```suggestion
#ifndef __SWITCH__
    BootCommands_ParseBootArgs(argc - 1, (char**)&argv[1]);
#endif
```

Reply:
This is a leftover from soh n64, it causes a crash because it tries to clean null args